src/theme.js

```diff
@@ -1,5 +1,6 @@
 export default {
   colorPrimary: '#1385e5',
+  colorSecondary: '#e77213',
```

Comment:
I like the color :smile:

Reply:
sweet! I just did a 180 degree hue shift haha